src/function/matrix/eigs.js

```diff
@@ -47,14 +59,17 @@ export const createEigs = /* #__PURE__ */ factory(name, dependencies, ({ config,
    *     const UTxHxU = multiply(transpose(U), H, U) // diagonalizes H if possible
    *     E[0] == UTxHxU[0][0]  // returns true always
    *
+   *     // Compute only approximate eigenvalues:
+   *     const {values} = eigs(H, {eigenvectors: false, precision: 1e-6})
+   *
    * See also:
    *
    *     inv
    *
    * @param {Array | Matrix} x  Matrix to be diagonalized
    *
-   * @param {number | BigNumber} [prec] Precision, default value: 1e-15
-   * @return {{values: Array|Matrix, eigenvectors: Array<EVobj>}} Object containing an array of eigenvalues and an array of {value: number|BigNumber, vector: Array|Matrix} objects.
+   * @param {number | BigNumber | OptsObject} [opts] Object with keys `precision`, defaulting to config.epsilon, and `eigenvectors`, defaulting to true and specifying whether to compute eigenvectors. If just a number, specifies precision.
```

Comment:
Can you add a unit test that tests the new feature of specifying a precision via an object?

Reply:
Yes, there are a couple now, because codecov also necessitated one with a matrix and an obj. And I had to remove newly-dead code in complexEigs, too. So I think/hope all is good to go now.

Reply:
Thanks! I made one comment on the unit test.

And one more thing: can you update the TypeScript type definitions of `eigs`?

Reply:
Yes, sorry about the oversight of not changing the typescript to match the new options. Fixed now.

Reply:
Thanks! Well, forgetting to update the types is not really your fault. We really need to have these types generated automatically 😉.